packages/code-du-travail-data/dataset/dossiers/index.js

```diff
@@ -8,146 +6,402 @@ const thematicFiles = [
-
-Déclarer un arrêt de travail pour un salarié contraint de garder son enfant à domicile ou dit à « risque » (affections longue maladie etc) :
- - https://declare.ameli.fr/
-
-Coronavirus et paiement des cotisations salariales et patronales :
- - https://www.urssaf.fr/portail/home/actualites/toute-lactualite-employeur/mesures-exceptionnelles-pour-les.html
-
-Coronavirus et données personnelles :
- - https://www.cnil.fr/fr/coronavirus-covid-19-les-rappels-de-la-cnil-sur-la-collecte-de-donnees-personnelles
-`,
-    refs: [
-      {
-        type: "main",
-        url:
-          "/dossiers/ministere-du-travail-notre-dossier-sur-le-coronavirus/ce-que-changent-les-ordonnances-en-droit-du-travail",
-        title:
-          "Coronavirus : ce que changent les ordonnances en droit du travail",
-        description:
-          "Le gouvernement a décidé de prendre plusieurs mesures afin d’accompagner les entreprises et les salariés pour faire face aux conséquences de la crise du coronavirus",
-        skipHydration: true,
-        source: SOURCES.THEMATIC_FILES,
-      },
-      {
-        type: "main",
-        url: "/fiche-ministere-travail/questions-reponses-par-theme",
-      },
-      {
-        type: "secondary",
-        url: "/fiche-ministere-travail/activite-partielle-chomage-partiel",
-      },
-      {
-        type: "secondary",
-        url:
-          "/fiche-service-public/activite-partielle-demarches-de-lemployeur-chomage-partiel-ou-technique",
-      },
-      {
-        type: "secondary",
-        url:
-          "/fiche-service-public/remuneration-dun-salarie-place-en-activite-partielle-chomage-partiel",
-      },
-      {
-        type: "secondary",
-        url:
-          "/fiche-ministere-travail/fiches-conseils-metiers-et-guides-pour-les-salaries-et-les-employeurs",
-      },
-      {
-        type: "secondary",
-        url: "/fiche-service-public/teletravail-dans-le-secteur-prive",
-      },
+    categories: [
       {
-        type: "secondary",
-        url:
-          "/fiche-ministere-travail/securite-et-sante-des-travailleurs-les-obligations-generales-de-lemployeur-et-sa-responsabilite",
-      },
-      {
-        type: "secondary",
-        url:
-          "/fiche-service-public/arret-maladie-indemnites-journalieres-versees-au-salarie",
-      },
-      {
-        type: "secondary",
-        url:
-          "/fiche-ministere-travail/formation-professionnelle-des-salaries-en-activite-partielle#Elargissement-du-dispositif-FNE-Formation-a-l-ensemble-des-entreprises-qui-nbsp",
-      },
-      {
-        type: "secondary",
-        url:
-          "/fiche-ministere-travail/mise-a-disposition-temporaire-de-salaries-volontaires-entre-deux-entreprises",
+        icon: "Populars",
+        id: "populaires",
+        position: 1,
+        refs: [
+          {
+            url:
+              "/modeles-de-courriers/attestation-sur-lhonneur-arret-de-travail-pour-la-garde-denfant",
+            title:
```

Comment:
I'm wonder if we should to this or only keep the url and resolve it while requesting it

Reply:
We have custom titles here in a lots of cases so they would resolve with different title and to avoid that we would have to make thing more complicated than needed IMO

Reply:
oh I did'nt notite that title here are not the same as the one of our document....
make sens :) 
on the other hand, it's not a bit weird to click on link with a given text that is different than the one on the page we land ?

Reply:
They don't differ that much but since the title is in a specific context, I guess slightly changing it helps in some case. Mostly when we split a fiche MT into chunks with long category titles